shared/settings/files/index.js

```diff
@@ -71,12 +175,30 @@ const styles = Styles.styleSheetCreate({
   contentHeader: {
     paddingBottom: Styles.globalMargins.tiny,
   },
-  mainContent: {
+  divider: {
+    marginTop: Styles.globalMargins.medium,
+  },
+  finderIntegrationContent: {
     paddingLeft: Styles.globalMargins.xsmall,
     paddingTop: Styles.globalMargins.medium,
   },
   spinner: {
     height: 16,
     width: 16,
   },
+  syncContent: {
+    paddingLeft: Styles.globalMargins.xsmall,
+    paddingTop: Styles.globalMargins.medium,
+  },
+  syncNotificationCheckbox: {
+    alignItems: 'center',
+  },
+  syncNotificationDropdownItem: {
+    width: 2 * Styles.globalMargins.xlarge,
```

Comment:
why is this so big?

Reply:
It's actually pretty small, that's how we get the root dropdown to look so small.